### ISSUE TITLE
T4 Duda número de filas de Q-Table

### ISSUE THREAD

**Issue by pedrodepabloi** [IIC2613-Inteligencia-Artificial-2022-2/Syllabus#87]
Hola!
Estaba viendo la inicialización de la Q-Table, y en el enunciado dice que debe tener 1.599.936 filas. Pero haciendo la matemática, me da que deberían ser 159.936 filas (51 x 28 x 2 x 28 x 2). ¿Puede ser que el número del enunciado esté incorrecto?
Muchas gracias!

**Comment by dfloreaa** [comment]
Efectivamente, sin querer cometí un error a la hora de escribir el número en el enunciado, gracias por notificarlo :)
